doc/examples/charts/bar-facing.py

```diff
@@ -37,6 +38,7 @@
 data = {
   "Hobbies" : ["Archery", "Tennis", "Football", "Basket", "Volley", "Golf", "Video-Games", "Reading", "Singing", "Music"],
   "Female" : proportions_female,
+  # Negate these values so they appear to the left side
   "Male" : -proportions_male
```

Comment:
I didn't know you could negate a list !

Reply:
That's not a list, but a NumPy array... 😉